README.md

```diff
@@ -3,8 +3,8 @@
 ## About
 
 This repository defines how to store geospatial [vector data](https://gisgeography.com/spatial-data-types-vector-raster/) (point, 
-lines, polygons) in [Parquet](https://parquet.apache.org/), a popular columnar storage format for tabular data - see 
-[what is parquet?](https://databricks.com/glossary/what-is-parquet ) for more background. Our goal is to standardize how 
+lines, polygons) in [Apache Parquet](https://parquet.apache.org/), a popular columnar storage format for tabular data - see 
+[what is parquet?](https://parquet.apache.org/docs/overview/) for more background. Our goal is to standardize how 
```

Comment:
[this vendor explanation](https://databricks.com/glossary/what-is-parquet) for more on what that means. Our goal is to standardize how